querydsl-jpa/src/main/java/com/querydsl/jpa/JPAListAccessVisitor.java

```diff
@@ -28,10 +28,13 @@
 
     private final QueryMetadata metadata;
 
+    private final Map<Expression<?>, Path<?>> aliases;
+
     private final Map<Path<?>, Path<?>> replacements = Maps.newHashMap();
 
-    public JPAListAccessVisitor(QueryMetadata metadata) {
+    public JPAListAccessVisitor(QueryMetadata metadata, Map<Expression<?>, Path<?>> aliases) {
         this.metadata = metadata;
+        this.aliases = aliases;
```

Comment:
Shouldn't these be used?
I don't see a method similar to `JPAMapAccessVisitor.shorten`.